test/blackbox/docs/dummy.json

```diff
@@ -47,6 +47,11 @@
       "$ref": "#/definitions/dummyObject"
     }
   },
+  "patternProperties": {
+    "S_pattern": {
+      "type": "string"
+    }
```

Comment:
Please try to render the pattern like `^S(.?*)pattern&`, because `S_pattern` is a simple "regex".

Reply:
Still renders it as `private _sPatternPatternProperty?: Map<String, string>;`